transient/transient.py

```diff
@@ -46,6 +54,74 @@ def __needs_ssh_console(self) -> bool:
                 self.config.ssh_with_serial is True or
                 self.config.ssh_command is not None)
 
+    def __get_required_environment(self) -> Environ:
+        """Configures the environment to remove the libvirt dependency from libguestfs-tools"""
+        env = os.environ
+        env['LIBGUESTFS_BACKEND'] = 'direct'
```

Comment:
Haha, nice. I was thinking that should just be in a common function. Unfortunate about the Environ stuff. I'm increasingly not a fan of mypy